specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -3101,6 +3101,39 @@
         "message"
       ]
     },
+    "VariableDefinitionSpecification": {
+      "description": "Definition of all variables for an Pipeline.",
```

Comment:
"a Pipeline"

Additionally - this description says "Definition of *all* variables - yet it only references a single Specification.
Should it be referencing an array or `VariableSpecifications`? If not - maybe alter this description.